components/Globals/FormField.tsx

```diff
@@ -31,21 +45,40 @@ const TextArea = ({ name, label }) => {
   );
 };
 
-const TextField = ({ name, label }) => {
-  return (
-    <div className="w-full my-2 leading-4">
-      <label className="w-full block text-left my-2" htmlFor={name}>
-        {label}
-      </label>
-      <input
-        className="bg-gray-600 p-1 w-full mb-2 rounded-md
+const TextField = ({ name, label, type, notRequired }) => {
+  type = type ? type : "text";
+  if (notRequired) {
```

Comment:
Is there any difference between these two components returned by the if? If the only difference is `required`, you can just set `required={required}` and get rid of the if